### DISCUSSION TITLE
thunderbird progress bars and status bars

### DISCUSSION THREAD

**Discussion by smeveriss** [nvaccess/nvda#13856] [General]
Hi.
The actual bug to thunderbird is here if anyone wants to read it.
https://bugzilla.mozilla.org/show_bug.cgi?id=1777184

So basically in a nut shell something has changed in thunderbird where progress bars for send and recieve message do not automatically announce themselves.
This is for thunderbird 102 stable.
Also with the new spaces bar, when navigating to it with statusbar is on status bar is announced and I'd like that extra info of the words status bar suppressed.
To be honest I'd really like the ability to take the spaces menu out of the tab order and toolbar tab order completely so I never need to see it again but there aint a will to do that.

Its not a big issue  in the reguard that its a big issue or showstopper but its an extra tab over something I can't turn off and I'd really like it sorted if at all possible.
The progress bar thing is probably the biggest thing.
On nvda list it was mentioned that mozilla enhancements could be modified for this.
However I have not needed to use any mozilla addons for nvda for firefox or for thunderbird and don't see the need myself.
But if thats the easiest way to keep things working then so be it I guess.
Its just this thing worked before and now it doesn't.
Technically I could revert back to 91.1 if I really had to but I see 2 issues with this.
1.  it would become like when I used waterfox, I could do that for a time but eventually I'd need to decide where to go.
2.  more importantly with various things shifting with cloud security I wouldn't be surprised if google, etc decided to shift their auth methods again so keeping updated is really important.
Pluss the program has improved and while I need to get used to some new tabs and nvda flipping in and out of forms mode as it cycles between all sorts of fields in settings I do like it a lot, certainly more than the quibbles I currently have with it.

**Comment by lukaszgo1** [comment]
@smeveriss  This issue cannot be triaged as it is. Please try to follow one of the issue templates, provide steps to reproduce (written in a way in which someone not using Thunderbird daily can follow them), explain the current behavior and what  you want to happen instead. Also since this apparently affects both status bar and some progress bar these would be best submitted as separate issues.
